autoload/asyncomplete/sources/buffer.vim

```diff
@@ -26,7 +26,7 @@ endfunction
 
 function! asyncomplete#sources#buffer#get_source_options(opts)
     return extend({
-        \ 'events': ['BufWinEnter'],
+        \ 'events': ['BufWinEnter', 'TextChangedI', 'InsertLeave'],
```

Comment:
We need to debounce to avoid perf issues if we want to listen to TextChangedI or TextChangedP.